collections/ansible_collections/massopencloud/esi/roles/agent/tasks/import_agents.yaml

```diff
@@ -0,0 +1,186 @@
+- name: Display node names that will be added as agents
+  ansible.builtin.debug:
+    var: node_names
+
+- name: Get list of existing agents
+  kubernetes.core.k8s_info:
+    kind: Agent
+    api_version: agent-install.openshift.io/v1beta1
+    namespace: "{{ agent_namespace }}"
+  register: esi_initial_agents
+
+- name: Initialize esi_node_info_list and esi_node_addresses_list
+  ansible.builtin.set_fact:
+    esi_node_info_list: []
+    esi_node_addresses_list: []
```

Comment:
I don't think you need this initialization, since you're setting the values directly below.